neurons/validators/utils/uids.py

```diff
@@ -52,41 +54,40 @@ def get_top_miner_uids(
     """
 
     dendrite = bt.dendrite(wallet=wallet)
-    try:
-        miner_candidate_uids = []
-        for uid in range(metagraph.n.item()):
-            uid_is_available = check_uid_availability(
-                metagraph, uid, vpermit_tao_limit
-            )
-
-            if uid_is_available:
-                miner_candidate_uids.append(uid)
-
-        miner_healthy_uids, _ = await ping_uids(dendrite, metagraph, miner_candidate_uids)
-
-        ips = []
-        miner_ip_filtered_uids = []
-        for uid in miner_healthy_uids:
-            if metagraph.axons[uid].ip not in ips:
-                ips.append(metagraph.axons[uid].ip)
-                miner_ip_filtered_uids.append(uid)
+    miner_candidate_uids = []
+    for uid in range(metagraph.n.item()):
+        uid_is_available = check_uid_availability(
+            metagraph, uid, vpermit_tao_limit
+        )
 
-        # Consider both of incentive and trust score
-        values = [(uid, metagraph.I[uid] * metagraph.trust[uid]) for uid in miner_ip_filtered_uids]
+        if uid_is_available:
+            miner_candidate_uids.append(uid)
 
-        # Consider only incentive
-        # values = [(uid, metagraph.I[uid]) for uid in candidate_uids]
+    miner_healthy_uids, _ = await ping_uids(dendrite, metagraph, miner_candidate_uids)
 
     ips = []
-    filtered_uids = []
-    for uid, _ in sorted_values:
+    miner_ip_filtered_uids = []
+    for uid in miner_healthy_uids:
         if metagraph.axons[uid].ip not in ips:
             ips.append(metagraph.axons[uid].ip)
-            filtered_uids.append(uid)
+            miner_ip_filtered_uids.append(uid)
+
+    # Consider both of incentive and trust score
+    values = [(uid, metagraph.I[uid] * metagraph.trust[uid]) for uid in miner_ip_filtered_uids]
+
+    # # Consider only incentive
```

Comment:
Does any of this code still apply? Should it be commented or removed?

Reply:
it will be removed soon

Reply:
removed